src/config.rs

```diff
@@ -348,7 +355,7 @@ pub async fn remappings_txt(
 ) -> Result<()> {
     let remappings_path = get_current_working_dir().join("remappings.txt");
     if soldeer_config.remappings_regenerate {
-        remove_file(&remappings_path).map_err(ConfigError::RemappingsError)?;
+        let _ = remove_file(&remappings_path);
```

Comment:
I'm interested to know why you ignore any error here

Reply:
because if the remapping does not exists then this should not revert. E.g. i set the regenerate = true first time, this will fail.

Reply:
Ok this is not ideal, a check should be made to check whether the file exists prior to trying to remove it, and any error in doing so (e.g. if permissions don't allow the removal) should be raised appropriately

Reply:
i wanted to add the check here to see if the file exists but i think i moved to other stuff :D will do it before release.